pkgs/applications/graphics/akira/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, lib, fetchFromGitHub
+, desktop-file-utils, meson, ninja, pkgconfig, vala_0_40, wrapGAppsHook
+, appstream-glib, cairo, glib, goocanvas2, gtk3, gtksourceview3
+, json-glib, libarchive, libgee, libxml2, pantheon, python3 }:
+
+stdenv.mkDerivation rec {
+  pname = "akira";
+  version = "2019-10-12";
+
+  src = fetchFromGitHub {
+    owner = "akiraux";
+    repo = "Akira";
+    rev = "cab952dee4591b6bde34d670c1f853f5a3ff6b19";
+    sha256 = "1fp3a79hkh6xwwqqdrx4zqq2zhsm236c6fhhl5f2nmi108yxz04q";
+  };
+
+  nativeBuildInputs = [ desktop-file-utils meson ninja pkgconfig vala_0_40 wrapGAppsHook ];
+  buildInputs = [
+    appstream-glib cairo glib goocanvas2 gtk3 gtksourceview3
+    json-glib libarchive libgee libxml2 meson pantheon.granite python3
```

Comment:
I don't think meson is going to be a runtime dependency here. No other references to python3 too
* https://github.com/akiraux/Akira/search?q=python3&unscoped_q=python3

Reply:
`appstream-glib` is just used for appstream validation. it's native.